### PR TITLE
Restricted unlogged users from uploading files

### PR DESCRIPTION
**Pull request by kristenking**
added loggedIn state to track user login status. LoggedIn state is passed to UploadModal as a prop. UploadModal conditionally renders upload form or "please log in" message based on the loggedIn prop.

### CODE REVIEW

File: client/src/components/LoginModal.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
   Input,
 } from "@chakra-ui/react";
 
-const LoginModal = ({ loginModalOpen, openLoginModal, closeLoginModal, openSignupModal }) => {
+const LoginModal = ({ loginModalOpen, closeLoginModal, openSignupModal, setLoggedIn }) => {
```
Comment (vmaineng) [inline client/src/components/LoginModal.js:16]: Hey @kristenking ; It seems openLoginModal does not exist within your changes. I'm thinking it should be added. Please let me know what your thoughts are. Thanks!

**Review by thomas065 — APPROVED** [review]
Great work!

File: client/src/components/LoginModal.js (hunk 1 above)
Comment (kristenking) [reply client/src/components/LoginModal.js:16]: @vmaineng openLoginModal is not used anywhere in our code and looks like it is redundant that is why it was removed. there is loginModalOpen carries the function to open our login model. Please, let me know if you have any diffrent information or seen where it is used.

Comment (vmaineng) [reply client/src/components/LoginModal.js:16]: @kristenking it seems openLoginModal is being used in line 62 in your code as a function. Should it be deleted?

**Review by vmaineng — APPROVED** [review]
Thanks for clarifying and explaining it's not being used in the code! Please feel free to merge :)

**Pull request merged**